templates/js/copilot-plugin-from-scratch/env/.env.local

```diff
@@ -4,8 +4,6 @@
 TEAMSFX_ENV=local
 
 # Generated during provision, you can also add your own variables.
-TAB_DOMAIN=
-TAB_ENDPOINT=
 TEAMS_APP_ID=
 TEAMS_APP_PACKAGE_PATH=
 FUNC_NAME=
```

Comment:
Seems func name is not used in the code, right? If so, can be removed as well.

Reply:
Yeah, removed.